views/add_event.hbs

```diff
@@ -214,9 +214,24 @@
           <input type="text" name="place_name_en" placeholder="English">
           <input type="text" name="place_name_ar" placeholder="Arabic">
         </label>
-      </div>
       <p class="msg-box-place error-msg"></p>
+
+      <label>Address
+        <input type="test" name="placeAddress" required>
+      </label>
+      <label>Website / Facebook Link
+        <input type="text" name="placeWebsite">
+      </label>
+
+      <label>Email
+        <input type="text" name="placeEmail">
+      </label>
+
+      <label>Phone number
+        <input type="text" name="placePhone">
```

Comment:
Tempted to say phone should be required, but I guess for now we can just leave that up to the location owners :man_shrugging: